Web/src/components/games/PandaSequence/GameDisplay.tsx

```diff
@@ -35,6 +37,26 @@ const GameContainer = styled.div`
   text-shadow: 3px 3px 0px rgba(0, 0, 0, 0.1);
 `;
 
+/**
+ * Wrapper to remove custom DOM attributes before rendering HTML DOM
+ * See: https://www.styled-components.com/docs/faqs#why-am-i-getting-html-attribute-warnings
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const BalloonSVGElement = ({ duration, isselected, ...props }:any) => (<BalloonSVG {...props} />);
```

Comment:
Some suggestions to be more type-safe:

```typescript
interface BalloonSVGElementProps extends React.ComponentProps<typeof BalloonSVG> {
  duration?: number,
  isSelected?: boolean,
}

/**
 * Wrapper to remove custom DOM attributes before rendering HTML DOM
 * See: https://www.styled-components.com/docs/faqs#why-am-i-getting-html-attribute-warnings
 */
// eslint-disable-next-line @typescript-eslint/no-unused-vars
const BalloonSVGElement = ({ duration, isSelected, ...props }: BalloonSVGElementProps) => (<BalloonSVG {...props} />);


interface DisplayBalloonProps extends BalloonSVGElementProps {
  duration: number,
}

interface InputBalloonProps extends BalloonSVGElementProps {
  isSelected: boolean,
}

const DisplayBalloon = styled(BalloonSVGElement)`
  width: 70px;
  margin: 12px;
  animation: ${({ duration }: DisplayBalloonProps) => (duration === 0
    ? undefined : css`${duration / 1000}s ${keyframes`${wobble}`} ease-in-out infinite;`)}
`;

const InputBalloon = styled(BalloonSVGElement)`
  width: ${({ isSelected }: InputBalloonProps) => (isSelected ? '100px' : '70px')}
  margin: 12px;
`;
```

correspondingly change the `isselected` below to `isSelected`.

This works because we are already removing `isSelected` from things to be passed to the React DOM svg element

Reply:
Yeah, but we are already filtering it out before passing it in? I tried and it works...

Note `{ duration, isSelected, ...props}`

So `props` will never contain `isSelected`, thus the React part handling DOM element that is BalloonSVG should not even be aware that `isSelected` exists?

Reply:
Right I'm not sure what I was doing there, I must have missed something because it works now! I somehow thought that the JSX element had to follow the same rule 😅